pages/linux/termusic.md

```diff
@@ -0,0 +1,12 @@
+# termusic
+
+> A terminal music player written in Rust that uses vim-like key bindings.
+> More information: <https://github.com/tramhao/termusic>.
+
+- Open termusic to a specific directory. **This can be set permanantly in `~/.config/termusic/config.toml`.**:
+
+`termusic <PATH/TO/MUSIC>`
+
+- View termusic's usage info:
```

Comment:
```suggestion
- Disable the album cover:

`termusic -c {path/to/music_file}`
- View termusic's usage info:
```
I think adding this flag would be useful too 
![image](https://user-images.githubusercontent.com/26346867/183554908-b7517123-c523-4f7a-b1d4-edb8382e6f54.png)